src/components/feed/index.js

```diff
@@ -0,0 +1 @@
+export * as Feed from "./feed";
```

Comment:
I didn't work on feed - do you want me to change it for the repo?

Reply:
Yeah sorry- I just saw this because it's showing as a change in this PR; if you can just make that one small change in this PR it will save us having to make another one later 👍

Reply:
done